content/08.transmission.md

```diff
@@ -284,3 +284,40 @@ COVID-19 can thus be considered a "syndemic", or a synergistic interaction betwe
+
+Another important question is whether patients with moderate to severe cases are able to access hospital facilities and treatments, to the extent that they have been identified.
+Early findings from China as of February 2020 suggested the COVID-19 mortality rate to be much lower in the most developed regions of the country [@doi:10/ggqscd].
+Efforts to make treatment accessible for all confirmed and suspected cases of COVID-19 in China are credited with expanding care to people with fewer economic resources [@doi:10.1038/s41591-020-0823-6].
+In the United States, access to healthcare varies widely, with certain sectors of the workforce less likely to have health insurance; many essential workers in transportation, food service, and other frontline fields are among those likely to be uninsured or underinsured [@doi:10.1016/j.jaad.2020.04.046].
+As of 2018, Hispanic Americans of all races were much less likely to have health insurance than people from non-Hispanic backgrounds [@url:https://www.census.gov/content/dam/Census/library/publications/2019/demo/p60-267.pdf].
+Therefore, access to diagnostics and care prior to the development of severe COVID-19 is likely to vary depending on socioeconomic and social factors, many of which overlap with the risks of exposure and of developing more severe COVID-19 symptoms.
+This discrepancy ties into concerns about broad infrastructural challenges imposed by COVID-19.
+A major concern in many countries has been the saturation of healthcare systems due to the volume of COVID-19 hospitalizations (e.g., [@doi:10.1073/pnas.2004064117]).
+Similarly, there have been shortages of supplies such as ventilators that are critical to the survival of many COVID-19 patients, leading to extensive ethical discussions about how to allocate limited resources among patients [@doi:10.19044/esj.2020.v16n21p24; @doi:10.7249/PEA228-1; @arxiv:2008.00374; @doi:10.1016/j.surg.2020.04.044].
+Although it is generally considered unethical to consider demographic factors such as age, sex, race, or ethnicity while making such decisions, and ideally this information would not be shared with triage teams tasked with allocating limited resources among patients [@doi:10.1080/15265161.2020.1764137], there are substantial concerns about implicit and explicit biases against older adults [@doi:10.1111/jgs.16539], premature infants [@doi:10.1111/apa.15334], and people with disabilities or comorbidities [@doi:10.1080/15265161.2020.1764138; @doi:10.2105/AJPH.2020.305837; @doi:10.1080/15265161.2020.1764137].
+Because of the greater burden of chronic disease in populations subjected to systemic racism, algorithms intended to race and ethnicity could, in fact, reinforce systemic inequalities caused by structural racism [@doi:10.1080/15265161.2020.1777347; @doi:10.1016/j.jnma.2020.05.010; @doi: 10.1056/NEJMp2023616].
```

Comment:
```suggestion
Because of the greater burden of chronic disease in populations subjected to systemic racism, algorithms intended to be blind to race and ethnicity could, in fact, reinforce systemic inequalities caused by structural racism [@doi:10.1080/15265161.2020.1777347; @doi:10.1016/j.jnma.2020.05.010; @doi: 10.1056/NEJMp2023616].
```

Words help!